common/src/main/kotlin/io/github/techtastic/cc_vs/util/CCVSUtils.kt

```diff
@@ -20,10 +22,26 @@ object CCVSUtils {
         if (ship == null)
             return
 
+        // Get the 'Computer' instance from 'computer' (which is an instance of ServerComputer)
```

Comment:
We have a [public API function](https://tweaked.cc/mc-1.20.x/javadoc/dan200/computercraft/api/ComputerCraftAPI.html#registerAPIFactory(dan200.computercraft.api.lua.ILuaAPIFactory)) for this. You don't need to mixin or use reflection for this.